modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/graphql/validation/GraphQLRequestContextValidator.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.vulcan.graphql.validation;
+
+/**
+ * @author Carlos Correa
+ */
+public interface GraphQLRequestContextValidator {
+
+	public void clear() throws Exception;
```

Comment:
Is this method necessary? I don't see you're calling it anywhere. It would be much cleaner if this method were not in the interface.

Reply:
It is being called from the GraphQLServletExtender class when the invocation to the servlet has finished

Reply:
Hmm I don't see it in GitHub. I'll download the PR.

Reply:
Ok, I will remove it and will clean the scopeContext directly by accessing its method